webapp/src/app/pacman-features/secondary-components/pacman-policy-violations/pacman-policy-violations.component.ts

```diff
@@ -123,8 +123,16 @@ export class PacmanPolicyViolationsComponent implements OnInit, OnDestroy {
   columnsSortFunctionMap = {
     Status: (a, b, isAsc) => {
       const order = ["fail", "exempt", "exempted", "pass"];
+      const severityOrder = ["critical", "high", "medium", "low"];
+      
       const AStatus = a["Status"].valueText.toLowerCase();
       const BStatus = b["Status"].valueText.toLowerCase();
+      const ASeverity = a["Severity"].valueText.toLowerCase();
+      const BSeverity = b["Severity"].valueText.toLowerCase();
+
+      if(order.indexOf(AStatus)==order.indexOf(BStatus)){
+        return (severityOrder.indexOf(ASeverity) < severityOrder.indexOf(BSeverity) ? -1 : 1) * (isAsc ? 1 : -1);
+      }
```

Comment:
You have order and severityOrder here as an array. But all you care about is index in that array. 
Why don't convert those two into maps? Keys - severity levels, values - their assigned number/weights.
So then you will not need to use `indexOf` to search exact index, you just need to get element from the map.
Check out other columnsSortFunctionMap functions and you'll see how to do that, AssetGroupsComponent for example.